aspnet/fundamentals/dependency-injection/sample/NuGet.Config

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Some commands ignore this file unless named exactly `NuGet.config` (lowercase 'c').